pkg/instancetype/instancetype.go

```diff
@@ -386,6 +386,10 @@ func checkCPUPreferenceRequirements(instancetypeSpec *instancetypev1beta1.Virtua
 		if vmiSpec.Domain.CPU.Sockets < preferenceSpec.Requirements.CPU.Guest {
 			return cpuField.Child("sockets"), fmt.Errorf(InsufficientVMCPUResourcesErrorFmt, vmiSpec.Domain.CPU.Sockets, preferenceSpec.Requirements.CPU.Guest, "sockets")
 		}
+	case instancetypev1beta1.PreferSpread:
+		if vmiSpec.Domain.CPU.Cores+vmiSpec.Domain.CPU.Sockets < preferenceSpec.Requirements.CPU.Guest {
+			return cpuField.Child("cores,sockets"), fmt.Errorf(InsufficientVMCPUResourcesErrorFmt, vmiSpec.Domain.CPU.Cores+vmiSpec.Domain.CPU.Sockets, preferenceSpec.Requirements.CPU.Guest, "cores and sockets")
```

Comment:
I'm not sure that `"cores,sockets"` is a valid way to express two fields here, do we have any other examples?

Reply:
I have changed the return value from `k8sfield.Path` to `Conflicts` so we can return more than one path. I have added it as a separate commit, so if you would like to do it some other way, I can drop it.